public/data/johan237.json

```
@@ -0,0 +1,28 @@
+{
+  "name": "Kemmegne Tchassem Johan",
+  "type": "personal",
+  "bio": "I love building new things and challenging the status quo!",
+  "avatar": "https://github.com/johan237.png",
+  "links": [
+    {
+      "name": "Follow me on GitHub",
+      "url": "https://github.com/johan237",
+      "icon": "github"
+    },
+    {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/johan_tchassem",
+      "icon": "twitter"
+    }
+  ],
+  "milestones": [
+    {
+      "title": "Started bloging",
+      "date": "April 2022",
+      "icon": "blog",
```

We don't have an icon for the keyword **`blog`**, instead, use **`hashnode`.** 

```suggestion
      "icon": "hashnode",
```